usagov/usagov-2021
# USAGOV-38-ckeditor-image-sizing

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-38

## Description

Enable Image Resizing in CKEditor
- Install and enable ckeditor_media_resize
- Uninstall google_docs_filter
- Turn on filter_resize_media for USWDS text format
- Turn on `Limit allowed HTML tags and correct faulty HTML` for USWDS text format
- Styles for resized or left/right aligned images

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


- Create a new Basic Page
- On the Body Field, click the 'Insert Media' button (the button with a photo icon + musical note icon)
- Select an image from the media library and click "Insert selected"
- Click on the inserted image and confirm that the resize handles appear on the corners
- Resize the image and note the width
- Save the page and view it
- Confirm that the image is displayed at the chosen width
- Confirm that the aspect ratio is correct and the image is not stretched
- Edit the page
- Resize the image again and note the new width
- Save the page and view it
- Confirm that the image is displayed at the new chosen width



### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
